components/automate-ui/src/app/modules/infra-proxy/create-databag-item-modal/create-databag-item-modal.component.ts

```diff
@@ -141,7 +139,7 @@ export class CreateDatabagItemModalComponent implements OnInit, OnDestroy {
       this.attr = JSON.parse(this.createForm.controls['itemAttr'].value.trim());
     }
     this.id = {'id': this.createForm.controls['itemId'].value.trim()};
-    const dataBagItem = {
+    const dataBagItem: DataBagItem = {
```

Comment:
Author note: Always, always, always strongly type. More often than not, just adding that will reveal errors in the data instance.